django/db/backends/sqlite3/introspection.py

```diff
@@ -57,6 +60,16 @@ def __getitem__(self, key):
 class DatabaseIntrospection(BaseDatabaseIntrospection):
     data_types_reverse = FlexibleFieldLookupDict()
 
+    def get_field_type(self, data_type, description):
+        field_type = super().get_field_type(data_type, description)
+        if description.pk and field_type in {'BigIntegerField', 'IntegerField'}:
+            # No support for BigAutoField as SQLite treats all integer primary
+            # keys as signed 64-bit integers:
+            # - https://www.sqlite.org/autoinc.html
+            # - https://www.sqlite.org/lang_createtable.html#rowid
```

Comment:
I'm not sure if we need these links in a comment :thinking: I would leave them in the ticket.